bindings/core/src/response.rs

```diff
@@ -308,6 +312,7 @@ pub enum Response {
     AddressesWithUnspentOutputs(Vec<AddressWithUnspentOutputs>),
     /// Response for:
     /// - [`MinimumRequiredStorageDeposit`](crate::method::AccountMethod::MinimumRequiredStorageDeposit)
+    /// - [`ComputeStorageDeposit`](crate::method::UtilsMethod::ComputeStorageDeposit)
     MinimumRequiredStorageDeposit(String),
```

Comment:
Does this need to be a String?

Reply:
Realistically probably not, but if the max supply is very high and someone sets the rent cost for a network also very high it could be larger than i64 and not be safe in JS anymore :trollface:

Reply:
yea, it returns an u64, so cannot be put in a number as is unfortunately